### ISSUE TITLE
Error on exit

### ISSUE THREAD

**Issue by imkebe** [Tribler/tribler#2042]
#### Please, fill all relevant items:
- [x] I have read CONTRIBUTING.md
- [x] I have tried with the latest pre-release version and I still can reproduce the issue.
##### Tribler version/branch+revision:

6.5.1
##### Operating system and version:

Windows 10
##### Expected behavior:

Exit smoothly
##### Actual behavior:

Error on exit
##### Steps to reproduce the behavior:

Close app
##### Relevant log file output:

```
Traceback (most recent call last):
  File "Tribler\Main\vwxGUI\settingsDialog.pyo", line 318, in saveAll
  File "Tribler\Main\vwxGUI\__init__.pyo", line 128, in invoke_func

  File "Tribler\Main\vwxGUI\MainFrame.pyo", line 648, in Restart
pywintypes.error: (2, 'FindExecutable', 'Nie mo\xbfna odnale\x9f\xe6 okre\x9clonego pliku.')
Exception in thread Notifier-timer-channelcast:
Traceback (most recent call last):
  File "threading.pyo", line 810, in __bootstrap_inner
  File "threading.pyo", line 1082, in run
  File "Tribler\Core\CacheDB\Notifier.pyo", line 104, in doQueue
  File "Tribler\Core\Utilities\twisted_utils.pyo", line 46, in callInThreadPool
  File "Tribler\Core\Utilities\twisted_utils.pyo", line 56, in isInThreadPool
AttributeError: 'NoneType' object has no attribute 'threads'
```

**Comment by whirm** [comment]
Thanks for the report!

**Comment by EL01** [comment]
Got the same error on exit 2 times in 6.5.0. will upgrade to 6.5.1 (win10/ASUSA75DE/8GB)

ERROR   1458728057.84 tunnel_community:724   
Direction must be either ORIGINATOR or EXIT_NODE
ERROR   1458728057.84             
log:443   

Unhandled Error
Traceback (most recent call last):
 
File "threading.pyo", line 763, in run 
File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner    
File "twisted\internet\base.pyo", line 1192, in run  
File "twisted\internet\base.pyo", line 1201, in mainLoop

--- <exception caught here> ---
 
File "twisted\internet\base.pyo", line 797, in runUntilCurrent  
File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data    
File "Tribler\community\tunnel\tunnel_community.pyo", line 1243, in crypto_in  
File "Tribler\community\tunnel\crypto\tunnelcrypto.pyo", line 81, in decrypt_str  
File "cryptography\hazmat\primitives\ciphers\base.pyo", line 174, in finalize  
File "cryptography\hazmat\backends\openssl\ciphers.pyo", line 137, in finalize

cryptography.exceptions.InvalidTag: 

Unhandled Error
Traceback (most recent call last):
  
File "threading.pyo", line 763, in run  
File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner  
File "twisted\internet\base.pyo", line 1192, in run  
File "twisted\internet\base.pyo", line 1201, in mainLoop

--- <exception caught here> ---  

File "twisted\internet\base.pyo", line 797, in runUntilCurrent  
File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data  
File "Tribler\community\tunnel\tunnel_community.pyo", line 1243, in crypto_in  
File "Tribler\community\tunnel\crypto\tunnelcrypto.pyo", line 81, in decrypt_str  
File "cryptography\hazmat\primitives\ciphers\base.pyo", line 174, in finalize  
File "cryptography\hazmat\backends\openssl\ciphers.pyo", line 137, in finalize

cryptography.exceptions.InvalidTag:

**Comment by whirm** [comment]
@EL01 It's not the same error and It should be fixed in 6.5.1.

Please create a new issue if it still happens after upgrading.

Thanks!
